jdi-light-mobile-examples/src/main/java/io/github/epam/enums/NavigationNums.java

```diff
@@ -0,0 +1,17 @@
+package io.github.epam.enums;
+
+/**
+ * Created by Roman_Iovlev on 3/2/2018.
+ */
+public class NavigationNums {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'NavigationNums' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)